src/appRouter.js

```diff
@@ -8,6 +8,7 @@ import { getToast, getOverlay } from './js/store/common/selectors';
 import LoadingOverlay from './js/common/overlay';
 import GlobalStyles from './js/hooks/globalStyles';
```

Comment:
if you dont need global styles you can get rid of it

Reply:
removed